src/components/Select/Select.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SelectWrapper, SelectDropdown } from './Select.styles';
+
+const Select = ({ value }) => {
+  return (
+    <SelectWrapper>
+      <label htmlFor="select">{value}</label>
+
+      <SelectDropdown name="select" id="select">
+        <option value="selectInstruction">Select an option...</option>
+        <option value="example1">1</option>
+        <option value="example2">2</option>
+        <option value="example3">3</option>
+      </SelectDropdown>
+    </SelectWrapper>
+  );
+};
+
+Select.propTypes = {
+  value: PropTypes.string.isRequired
+};
+
+export default Select;
```

Comment:
A few issues with this file: 
• Don't hard code the options in like that, pass them in as an array and map over them.
• I would change `SelectDropdown` to a `select` element and style it with a nested selector (see comments on Selectstyles.js).
• The `value` prop is not a value, rather the label text. Consider changing this `label`.
• Don't hard code the name of the select, pass it in as a prop called `name`.
• You need to pass in an `onChange` handler.
• Pass in a `defaultValue` prop.

So with the above in mind it should end up looking something like this:
```
const Select = ({ label, name, options, onChange, defaultValue }) => {
    return (
        <SelectWrapper>
            <label htmlFor={name}>{label}</label>
            <select
                name={name}
                id={name}
                onChange={onChange}
                value={defaultValue}
            >
                {options.map(({ value, label }, index)  =>  (
                     <option
                        key={value + index}
                        className="select--option"
                        value={value}
                       >
                          {label}
                     </option>
                ))}
            </select>
        </SelectWrapper>
    )
};

Select.propTypes = {
  name: PropTypes.string.isRequired,
  label: PropTypes.string.isRequired,
  defaultValue: PropTypes.string,
  options: PropTypes.arrayOf(
    PropTypes.shape({
      value: PropTypes.string.isRequired,
      label: PropTypes.string.isRequired,
    })
  ).isRequired,
  onChange: PropTypes.func.isRequired,
};
```